src/components/ProductDetail/ProductDetail.js

```diff
@@ -80,6 +79,8 @@ class ProductDetail extends Component {
       selectOptionId = variant.options[0]._id;
     }
 
+    this.trackAction({ variant, optionId, action: "Product Viewed" });
```

Comment:
We should make `Product Viewed` a constant and export from either `lib/tracking/constants.js` or from `lib/tracking/trackProduct.js`.